app/main.py

```diff
@@ -1 +1,53 @@
-# write your code here
+class SoftwareEngineer:
+    skills = []
```

Comment:
Skills should be an instance attribute not a class attribute.

https://mateacademy.slack.com/archives/C03ED3XARFV/p1653396979567459